gulpfile.js

```diff
@@ -117,3 +118,9 @@ gulp.task('build-js', function() {
   .bundle()
   .pipe(callback('static/js/common.js'));
 });
+
+gulp.task('test', function() {
+  return mochify('./tests/unit/**.js')
```

Comment:
I was thinking we'd keep dir organization here mirrored with the actual js. SO in this case, we'd have `tests/unit/modules` or `tests/unit/pages`. What do you think of that?